modules/services/views/src/main/java/au/com/shiftyjelly/pocketcasts/views/multiselect/MultiSelectFragment.kt

```diff
@@ -105,10 +121,72 @@ class MultiSelectFragment : BaseFragment(), MultiSelectTouchCallback.ItemTouchHe
     }
 
     override fun onItemStartDrag(viewHolder: MultiSelectAdapter.ItemViewHolder) {
+        dragStartPosition = viewHolder.bindingAdapterPosition
         itemTouchHelper.startDrag(viewHolder)
     }
 
-    override fun onItemTouchFinished() {
+    override fun onItemTouchFinished(position: Int) {
         settings.setMultiSelectItems(items.filterIsInstance<MultiSelectAction>().map { it.groupId })
+        trackItemMovedEvent(position)
+    }
+
+    private fun sectionTitleAt(position: Int) =
+        if (position < MultiSelectToolbar.MAX_ICONS) AnalyticsProp.Value.SHELF else AnalyticsProp.Value.OVERFLOW_MENU
```

Comment:
I think we need this to be `<=` instead of `<`. Without that change the events are acting like my shelf is one item smaller than it actually is (i.e., an item at the "end" of my shelf is treated as if it is in the overflow menu).

```suggestion
        if (position <= MultiSelectToolbar.MAX_ICONS) AnalyticsProp.Value.SHELF else AnalyticsProp.Value.OVERFLOW_MENU
```

Reply:
Oops, thanks for catching it 🙏